hack/release/main.go

```diff
@@ -0,0 +1,64 @@
+/*
+Copyright 2021 The Cockroach Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"regexp"
+
+	"sigs.k8s.io/kubetest2/pkg/process"
+)
+
+var (
+	dir     string
+	version string
+
+	deployedBundlesDir = filepath.Join("deploy", "certified-metadata-bundle", "cockroach-operator")
+	versionRegxp       = regexp.MustCompile(`^\d+\.\d+\.\d+(-beta\.\d+)?$`)
```

Comment:
nit: `deployedBundlesDir` appears unused?
nit: move `versionRegxp` to steps.go where it's used

Reply:
Nice catch! Before checking the tags, I was initially going to search the deploy directory. Removed.